_config.yml

```diff
@@ -23,4 +23,4 @@ url: 'https://impactlabs.io/'
 twitter:
   username: impactlabs_io
 
-logo: 'https://impactlabs.io/assets/images/social_media_banner.jpg'
+logo: 'https://impactlabs.io/assets/images/impact_labs.jpg'
```

Comment:
couldn't find social_media_banner.jpg

Reply:
Yeah this will work. To be honest, I think we should delete this line. The SEO plugin doesn't do anything with it

Reply:
```
logo - URL to a site-wide logo (e.g., /assets/your-company-logo.png) - If you would like the "publisher" property to be present, you must add this field to your site's configuration, during the validation of the structured data by Google Search Console, if the logo field is not validated, you will find errors inherent to the publisher in the Rich Results Testing Tool
```

the documentation seems to suggest something?